R/helpers.R

```diff
@@ -402,7 +405,8 @@ get_hover <- function(hn, data) {
   for(ii in seq_len(ncol(data)))
     data[[ii]] <- format(data[[ii]])
 
-  hdict <- lapply(seq_along(hn), function(ii) list(hn[ii], paste("@", hn2[ii], sep = "")))
+  hdict <- lapply(seq_along(hn), function(ii) list(gsub("hover_", "", hn[ii]), 
```

Comment:
I can't reproduce this anymore, so let's back it out.  I can't seem to update this PR anymore--my updates are going into the [data.table branch](https://github.com/jrowen/rbokeh/tree/data.table) that I have as clone of `hafen/rbokeh`.  I can submit a new PR to that project or you can feel free to back it out if you merge this PR.